src/main/java/dev/jbang/cli/Jbang.java

```diff
@@ -47,6 +47,12 @@ void setVerbose(boolean verbose) {
 		Util.setVerbose(verbose);
 	}
 
+	@Option(names = { "--quiet",
+			"-q" }, description = "jbang will not show any unnecessary messages.", scope = ScopeType.INHERIT)
```

Comment:
```suggestion
			"-q" }, description = "jbang will be quiet, only print when error occurs.", scope = ScopeType.INHERIT)
```

Reply:
also, shouldn't --verbose and --quiet be mutual exclusive ? 

see https://picocli.info/#_mutually_exclusive_options on how to do that in picoclil

Reply:
also, just "--quiet" as "--verbose" doesn't have "-v"

Reply:
ok